support-frontend/assets/pages/subscriptions-landing/components/subscriptionsLandingContent.tsx

```diff
@@ -32,11 +36,19 @@ function SubscriptionsLandingContent({
 				{subscriptionCopy.map((product, index) => (
 					<SubscriptionsProduct
 						title={product.title}
-						subtitle={product.subtitle ?? ''}
+						subtitle={
+							isBlackFriday && product.title === 'Guardian Weekly'
+								? 'Annual'
+								: product.subtitle ?? ''
+						}
 						description={product.description}
 						productImage={product.productImage}
 						buttons={product.buttons}
-						offer={product.offer}
+						offer={
+							isBlackFriday && product.title === 'Guardian Weekly'
```

Comment:
No need to repeat this condition, this could be pulled up to the top into a variable

Reply:
Thanks for the review, have added a function `validBlackFridayProduct` check as it needs product.title, is that what you mean?